src/applications/personalization/profile360/components/ProfileView.jsx

```diff
@@ -32,10 +40,13 @@ class ProfileView extends React.Component {
   };
 
   componentDidMount() {
+    const initializationTransaction = this.props.vet360InitializationTransaction;
     if (this.props.isVet360AvailableForUser) {
       this.props.fetchTransactions();
     } else {
-      // this.props.initializeUserToVet360()
+      if (!initializationTransaction || !isPendingTransaction(initializationTransaction) || !isSuccessfulTransaction(initializationTransaction)) {
+        this.props.initializeUserToVet360();
+      }
     }
```

Comment:
How come we need to check the `initializationTransaction` existence here? Asking because it's in  `componentDidMount` and it seems like it will always be `undefined` in the beginning.  

Not positive but I think we want to be working with the `vet360InitializationTransaction.transaction` property here instead of just `vet360InitializationTransaction` btw.